README.md

```diff
@@ -15,7 +15,9 @@ This repository contains the Canonical set of rules, or the Canonical Style.
 
 Anyone is welcome to submit a PR to add additional rules. However, no additions will be considered unless they are part of the Canonical Style Guide as found at the website above.
```

Comment:
```suggestion
Anyone is welcome to submit a PR to add additional rules. Additions can only be considered if they are part of (or conform to) the Canonical Style Guide as found at the website above.
```

Reply:
(personal preference, I like a positive phrasing)

Reply:
I think I may change my mind on this. There exists a case whereby it is valid to test for something ( e.g. repeated words ' the the' and such) which we are never going to write a specific rule for (RULE#83: Don't write stupid stuff) but are nevertheless valid things to test for. Maybe these should be an optional extra set or something.
In any case, I don't want to put people off from creating genuinely useful rules

Reply:
that would be okay with me.